src/main/java/eu/webeid/security/validator/AuthTokenV1Validator.java

```diff
@@ -0,0 +1,109 @@
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in all
+ * copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+
+package eu.webeid.security.validator;
+
+import eu.webeid.security.authtoken.WebEidAuthToken;
+import eu.webeid.security.certificate.CertificateLoader;
+import eu.webeid.security.exceptions.AuthTokenException;
+import eu.webeid.security.exceptions.AuthTokenParseException;
+import eu.webeid.security.validator.certvalidators.SubjectCertificateValidatorBatch;
+import eu.webeid.security.validator.ocsp.OcspClient;
+import eu.webeid.security.validator.ocsp.OcspServiceProvider;
+
+import java.security.cert.CertStore;
+import java.security.cert.TrustAnchor;
+import java.security.cert.X509Certificate;
+import java.util.Set;
+
+class AuthTokenV1Validator implements AuthTokenValidator {
+
+    private static final String SUPPORTED_TOKEN_FORMAT_VERSION = "web-eid:1";
+
+    private final SubjectCertificateValidatorBatch simpleSubjectCertificateValidators;
+    private final Set<TrustAnchor> trustedCACertificateAnchors;
+    private final CertStore trustedCACertificateCertStore;
+    private final AuthTokenSignatureValidator authTokenSignatureValidator;
+    private final AuthTokenValidationConfiguration configuration;
+    private final OcspClient ocspClient;
+    private final OcspServiceProvider ocspServiceProvider;
+
+    public AuthTokenV1Validator(
+        SubjectCertificateValidatorBatch simpleSubjectCertificateValidators,
+        Set<TrustAnchor> trustedCACertificateAnchors,
+        CertStore trustedCACertificateCertStore,
+        AuthTokenSignatureValidator authTokenSignatureValidator,
+        AuthTokenValidationConfiguration configuration,
+        OcspClient ocspClient,
+        OcspServiceProvider ocspServiceProvider
+    ) {
+        this.simpleSubjectCertificateValidators = simpleSubjectCertificateValidators;
+        this.trustedCACertificateAnchors = trustedCACertificateAnchors;
+        this.trustedCACertificateCertStore = trustedCACertificateCertStore;
+        this.authTokenSignatureValidator = authTokenSignatureValidator;
+        this.configuration = configuration;
+        this.ocspClient = ocspClient;
+        this.ocspServiceProvider = ocspServiceProvider;
+    }
+
+    @Override
+    public boolean supports(String format) {
+        return format != null && format.startsWith(SUPPORTED_TOKEN_FORMAT_VERSION);
```

Comment:
To make code more DRY, perhaps use a [template method](https://en.wikipedia.org/wiki/Template_method_pattern) `getSupportedFormatPrefix()` and implement it in `AuthTokenV1Validator` and `AuthTokenV11Validator`:

```suggestion
        return format != null && format.startsWith(getSupportedFormatPrefix());
    }

    protected String getSupportedFormatPrefix() {
        return V1_SUPPORTED_TOKEN_FORMAT_PREFIX;
```